.github/workflows/tests-integration.yaml

```diff
@@ -27,6 +27,8 @@ jobs:
         run: |
           uv sync --group tests --extra all
 
+      - run: python scripts/wake_up_hf_endpoint.py
```

Comment:
This calls with no retry to trigger the wake up but don't keep waiting while we install and run ollama